astropy/units/quantity_helper/function_helpers.py

```diff
@@ -469,7 +473,7 @@ def arange_impl(*args, start=None, stop=None, step=None, dtype=None, **kwargs):
     # bind positional arguments to their meaningful names
     # following the (complex) logic of np.arange
     match args:
-        case pos1,:
+        case (pos1,):
```

Comment:
Why this change?

Reply:
In this file I only replaced the two `fmt: off`/`fmt: on` pairs that RUF028 highlighted with `fmt: skip`. All the other changes were made automatically by Ruff formatter.

Reply:
I guess this is a ruff-format coming back to life.

For reference, the change is logically equivalent, but admittedly clearer: the comma by itself makes this expression a tuple, but most reader could think that it is a syntax error (and be confused that it runs), or  miss the comma entirely while reading and miss this branch for `case pos1` (which has the form of a catch-all `case`, which makes no sense as a first `case` since it would make all other branches unreachable).

Reply:
Last time I used such a thing, it was in C. So my brain cannot comprehend why it has to be a tuple with one element here. I find `case pos1` much easier to understand. 🤷‍♀️

Reply:
`case pos1:` means "match anything that can be binded to a variable named `pos1`. In other words, it matches anything.
`case (pos1,):` on the other hand roughly translates to "`if isinstance(x, Sequence) and len(x)==1`, then bind `x[0]` to `pos1`".
admittedly, Python 3.10's pattern matching takes some getting use to, and it's "brand new" for most of us who had to maintain compatibility with Python 3.9 until recently. I wouldn't have used it here if a classic `if/else` tree made the logic easier to parse.

Reply:
Oh... Indeed I did miss that trailing comma... 😆 😭 

Okay. The parenthesis is indeed better then. Thanks!